lib/Doctrine/DBAL/ConnectionRegistry.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL;
+
+use InvalidArgumentException;
+
+interface ConnectionRegistry
+{
+    /**
+     * Gets the default connection name.
+     *
+     * @return string The default connection name.
+     */
+    public function getDefaultConnectionName() : string;
+
+    /**
+     * Gets the named connection.
+     *
+     * @param string $name The connection name (null for the default one).
+     *
+     * @throws InvalidArgumentException in case the connection for the given name does not exist.
```

Comment:
should we throw a generic InvalidArgumentException or an exception defined in DBAL ?

Reply:
For now I used this generic exception similar to

https://github.com/doctrine/persistence/blob/master/lib/Doctrine/Persistence/AbstractManagerRegistry.php#L98

But we can also create a more specific `ConnectionNotFoundException` or so